custom/ewsghana/templates/ewsghana/ewsconfig.html

```diff
@@ -92,6 +92,24 @@
                 </div>
             </div>
         </div>
+        <div class="row-fluid">
+            <div class="span12">
+                <div class="btn-toolbar">
+                    <button class="btn" id="fix_locations">
+                        {% trans "Fix locations" %}
```

Comment:
I found problem when user is assigned to the location which is not administrative type (eg Region). 

Now, in this case we create Supply Point with location's external id which is not true because in EWS location id doesn't equal to supply point id.

My fix is only for existing EWS domain. Fix removes external ids from all not administrative locations and adds the new supply points for other not administrative locations, for that the situation will not occurred.